README.md

```diff
@@ -44,6 +44,15 @@ Screenshots
 
 Changelog
 ------------
+
+#### 1.5.3
+* Added support for requiring shipping and billing information
+* Test against PHP 7.0.10 
```

Comment:
@SenthilKumar-86 Would you address the other part of my previous comment to clarify that the plugin supports PHP 7?